helpers.py

```diff
@@ -17,8 +17,8 @@ def get_config_value(config, name, inverter_or_template, pvinverternumber, defau
         return config[f"{inverter_or_template}{pvinverternumber}"][name]
 
     if defaultvalue is None:
-        raise ValueError(f"config entry '{name}' not found. \
-            Hint: Deprecated Host ONPREMISE entries must be moved to DEFAULT section")
+        raise ValueError(f"config entry '{name}' not found. "
+                         f"(Hint: Deprecated Host ONPREMISE entries must be moved to DEFAULT section.)")
```

Comment:
It looks like there are spaces in front of line 21 but maybe it just looks like this

Reply:
Yes, but I think all our indentation is done by spaces, not tabs. And I prefer this because the indentation is ok even when the IDE (or vi or whatever) is not configured in the way we are expecting ist.